src/hooks/api/useCashFlow.ts

```diff
@@ -44,12 +44,25 @@ export function useCashFlow(
               JOIN accounts AS main_account ON main_account.guid = '${account}'
               WHERE commodities.guid = main_account.commodity_guid
             ) AS mnemonic,
-            SUM(((cast(main_split.quantity_num as REAL) / main_split.quantity_denom) / (cast(main_split.value_num as REAL) / main_split.value_denom)) * (cast(splits.value_num as REAL) / splits.value_denom)) as total
+            SUM(
+              CASE
+                WHEN splits.guid = credit_split.guid THEN
+                  -1 * cast(account_split.quantity_num AS REAL) / account_split.quantity_denom
```

Comment:
for the case where the split we are calculating for is the credit_split, we instead pick the quantity from the account_split and reference it to the credit account. An example:

```
Assets:Bank -100
Liabilities:L1 75
Expenses:E2 25
```

When we compute cashflow for L1 (cashflow is available for both assets and liabilities) we want it to be:

`Assets:Bank -75`